### PR TITLE
fix: try load official formatter

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case
When using formatter as string try loading the [official formatters](https://eslint.org/docs/user-guide/formatters/)
```js
{
  formatter: 'table' // https://eslint.org/docs/user-guide/formatters/#table
}
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -221,15 +206,41 @@ module.exports = function(input, map) {
...
+          config.formatter &&
+          typeof config.formatter !== "function" &&
+          typeof config.formatter.default === "function"
+        ) {
+          config.formatter = config.formatter.default;
+        }
+      } catch (_) {
+        // ignored
+      }
+    }
```
Comment (evilebottnawi) [inline index.js]: Please refactor this in nested try/catch, no need duplicate logic, also please add tests

File: test/formatter-oficial.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
```
Comment (evilebottnawi) [inline test/formatter-oficial.js]: Typo in file name

File: test/formatter-oficial.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      // console.log("### Here is a example of oficial formatter")
+      // console.log(
+      //   "# " +
+      //   stats.compilation.errors[0].message
+      //     .split("\n")
+      //     .join("\n# ")
+      // )
+      t.truthy(
+        stats.compilation.errors[0].message,
+        "webpack have some output with oficial formatter"
```
Comment (evilebottnawi) [inline test/formatter-oficial.js]: typo

File: test/formatter-oficial.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        throw err;
+      }
+
+      // console.log("### Here is a example of oficial formatter")
+      // console.log(
+      //   "# " +
+      //   stats.compilation.errors[0].message
+      //     .split("\n")
+      //     .join("\n# ")
+      // )
```
Comment (evilebottnawi) [inline test/formatter-oficial.js]: Need better test

**Comment by ricardogobbosouza** [comment]
Testing with formatters is like this! In other words, they are all incomplete, I'm going to launch a pr then improving all and putting coverage

File: test/formatter-oficial.js (hunk 4 above)
Comment (ricardogobbosouza) [reply test/formatter-oficial.js]: The tests with formatters are like this, in fact all tests needs to be improved, I'll do it in a next pr

File: test/formatter-official.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      //     .join("\n# ")
+      // )
+      t.truthy(
+        stats.compilation.errors[0].message,
+        "webpack have some output with official formatter"
+      );
+      t.end();
+    }
+  );
+});
```
Comment (evilebottnawi) [inline test/formatter-official.js]: oh, let's add test with `require`, just ensure it is works as expected

Comment (ricardogobbosouza) [reply test/formatter-official.js]: Already exists in `formatter-custom`

**Pull request merged**